root/types.js

```diff
@@ -108,12 +113,11 @@ declare type CommonsImageT = {|
   +thumb_url: string,
 |};
 
-declare type CompoundFieldT<F: {+[string]: mixed}> = {|
-  +errors: $ReadOnlyArray<string>,
-  +field: F,
-  +has_errors: boolean,
-  +id: number,
-|};
+declare type CompoundFieldT<F: {+[string]: AnyFieldT}> =
+  & FieldRoleT
+  & {|
+      +field: F,
```

Comment:
Shouldn't this also be changed according to PR #629 ? Same for the other types declared below.

Reply:
Yeah, will fix